tests/test_mod.py

```diff
@@ -502,6 +502,42 @@ def test_get_product_odata_short():
             assert ret[k] == expected[k]
 
 
+def scrub_string(string, replacement=''):
```

Comment:
Is there any reason to nest function definitions here? I would propose to change this to one function.

Reply:
Function definition nesting is used here to enable passing of `scrub_string` method as `before_record_response` callback. By default, callbacks need to be of format `callback_name(request)` (inferred from [vcrpy docs](https://vcrpy.readthedocs.io/en/latest/advanced.html#custom-response-filtering)). By nesting the `before_record_response` method the `scrub_string` methods allows scrubbing of _any_ string, not just hardcoded ones.

Could change this to a more specific `scrub_online_kv` callback?